scss/bootstrap/overrides/_variables.scss

```diff
@@ -86,6 +86,9 @@ $crm-accordion-header-color: $gray-lighter;
 $crm-darken-percentage:     7% !default;
 $crm-lighten-percentage:    30% !default;
 
+$crm-row-ok-background-color: #efffe7;
```

Comment:
question: why don't we use `$brand-warning` here? couldn't we have done something like `lighten($brand-warning, 10)` ?